front/app/containers/Admin/settings/registration/index.tsx

```diff
@@ -50,31 +50,55 @@ const SettingsRegistrationTab = (_props: Props) => {
   const [errors, setErrors] = useState<{ [fieldName: string]: CLError[] }>({});
   const [attributesDiff, setAttributesDiff] =
     useState<IUpdatedAppConfigurationProperties>({});
+  const [latestAppConfigSettings, setLatestAppConfigSettings] =
+    useState<IAppConfigurationSettings | null>(null);
+
+  useEffect(() => {
+    if (!isNilOrError(appConfig)) {
+      setLatestAppConfigSettings(appConfig.data.attributes.settings);
+    }
+  }, [appConfig]);
+
+  useEffect(() => {
+    setLatestAppConfigSettings((latestAppConfigSettings) => {
+      if (!isNilOrError(latestAppConfigSettings)) {
+        return {
+          ...latestAppConfigSettings,
+          ...attributesDiff.settings,
+        } as IAppConfigurationSettings;
+      }
+
+      return null;
+    });
+  }, [attributesDiff]);
```

Comment:
We now set the latestAppConfigSettings (a mix of the last saved settings + unsaved changes) via a hook.